src/bin/am/commands/start.rs

```diff
@@ -95,6 +95,10 @@ pub struct CliArguments {
     /// Whenever to clean up files created by Prometheus/Pushgateway after successful execution
     #[clap(short = 'd', long, env)]
     ephemeral: bool,
+
+    /// Whenever to *NOT* load the autometrics rules file into Prometheus
+    #[clap(long, env)]
+    no_rules: bool,
```

Comment:
It is good to keep `no_rules` here, we just need to flip the value so that we can use enable_rules in the rest of the application. Maybe there is some out of the box feature in clap that would help with this.

Reply:
i just pass it with a `!` to arguments